avaframe/com1DFA/com1DFA.py

```diff
@@ -267,8 +267,7 @@ def com1DFACore(cfg, avaDir, cuSimName, inputSimFiles, outDir):
     # +++++++++PERFORM SIMULAITON++++++++++++++++++++++
     # for timing the sims
     startTime = time.time()
-    particles, fields, dem, reportAreaInfo = initializeSimulation(cfg, demOri, inputSimLines, cuSimName)
-
+    particles, fields, dem, reportAreaInfo, demOri = initializeSimulation(cfg, demOri, inputSimLines, cuSimName)
```

Comment:
I think the logic for demOri vs dem needs to be tackled:
- If there is demOri as input, but an additional dem as output, I think the basic (and valid) assumption is, that demOri remains unchanged and `dem` contains the topography to use -> then there's no need to return demOri
- If demOri is somehow changed, why then the need for an additional `dem` variable?

I suggest to make it a clear distinction: demOri -> the originial, untouched DEM, dem -> the applied, and maybe modified, dem